contributing/background/background.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Background
+nav_order: 1
+has_children: true
+parent: Contributing
+---
+# Background Information
+
+## Getting Started
+
+To get started, we recommend the following resources (in order):
+
+1. [Schema.org Data Model](https://schema.org/docs/datamodel.html) (Data Commons adopts the Schema.org data model)
+1. [MCF Format](mcf_format.md)
+1. [Representing Statistics in Data Commons](representing_statistics.md)
+1. [Life of a Dataset](https://colab.sandbox.google.com/drive/1D_4IqrgT6SbbnNiOTQ16HzIG_T6KG1Wz?usp=sharing)
```

Comment:
should this link to your new markdown version? or was that not meant to be in this PR?

Reply:
My plan was for this PR to focus on doc organization and #61 to include the new markdown version of the original Colab notebook, so no, that updated link was not meant to be in this PR.

Reply:
I think there are also some diff's with #61, but we can merge this in and then rebase, then merge the other to get the complete set of changes. Since this includes that file, we can link to it right away.